config/packages/fos_elastica.yaml

```diff
@@ -106,6 +106,8 @@ fos_elastica:
                                     properties:
                                         id:
                                             type: 'keyword'
+                hasErddap:
+                    type: 'text'
```

Comment:
This doesn't return text.

Reply:
I wasn't sure if I could index a bool like that...good to know. I'll fix.